roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -51,8 +51,8 @@ volume_clear_size = {{ cinder.volume_clear_size }}
 glance_host = {{ endpoints.main }}
 glance_port = {{ endpoints.glance.port.backend_api }}
 
-{% if cinder.enabled_backends != None -%}
-enabled_backends = {{ cinder.enabled_backends }}
+{% if cinder.enabled_backends|length -%}
```

Comment:
I'd make this `cinder.enabled_backends|length => 1`. Unless you've tested to see that 0 == Boolean false, and > 1 == Boolean true.